### ISSUE TITLE
Add support for sorting

### ISSUE THREAD

**Issue by RickMoynihan** [OpenGovIntelligence/json-qb#22]
Format needs to define which dimension rows are sorted on and the direction it is sorted in ASC/DESC.

**Comment by zeginis** [comment]
In some cases lexicographical ordering is not appropriate. E.g. the days of the week should be Monday, Tuesday, Wednesday, Thursday ...

**Comment by BillSwirrl** [comment]
Yes that sounds a good suggestion.  In some of our other work we have added a triple with predicate <http://www.w3.org/ns/ui#sortPriority> to each concept in a concept scheme, then use that in the PublishMyData user interface to order items in different situations.  But I can see some options to return things ready-ordered in the API will be useful - especially if paging the results
